lib/Service/BailsPolicyProviderService.php

```diff
@@ -26,7 +26,11 @@ public function getPolicyForFile(File $file): ?IBailsPolicy {
 			if (!in_array($mimeType, $provider->getSupportedMimeTypes())) {
 				continue;
 			}
-			return $provider->getPolicyForFile($file);
+			try {
+				return $provider->getPolicyForFile($file);
+			} catch (\Exception) {
```

Comment:
Maybe catch all `Throwable`s?

```suggestion
			} catch (\Throwable) {
```